crates/sui-cluster-test/src/cluster.rs

```diff
@@ -0,0 +1,159 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+use super::config::{ClusterTestOpt, Env};
+use async_trait::async_trait;
+use clap::*;
+use sui_config::genesis_config::GenesisConfig;
+use sui_swarm::memory::Node;
+use sui_swarm::memory::Swarm;
+use sui_types::crypto::KeypairTraits;
+use sui_types::crypto::{get_key_pair, KeyPair};
+use test_utils::network::{start_rpc_test_network_with_fullnode, TestNetwork};
+
+const DEVNET_FAUCET_ADDR: &str = "https://faucet.devnet.sui.io:443";
+const STAGING_FAUCET_ADDR: &str = "https://faucet.staging.sui.io:443";
+const DEVNET_GATEWAY_ADDR: &str = "https://gateway.devnet.sui.io:443";
+const STAGING_GATEWAY_ADDR: &str = "https://gateway.staging.sui.io:443";
+const DEVNET_FULLNODE_ADDR: &str = "https://fullnode.devnet.sui.io:443";
+const STAGING_FULLNODE_ADDR: &str = "https://fullnode.staging.sui.io:443";
```

Comment:
@longbowlu can you add connectivity for the new continuous cluster endpoints please?

the endpoints are x.continuous.sui.io

Reply:
done